server/src/server.ts

```diff
@@ -30,5 +30,5 @@ const server = new ApolloServer({
 
 const httpServer = createServer(app);
 httpServer.listen({ port: PORT }, (): void =>
-  console.log(`🚀GraphQL-Server is running on http://localhost:3000/graphql`)
+  console.log(`🚀GraphQL-Server is running on http://localhost:z${PORT}/graphql`)
```

Comment:
Do you need the "z" in-front of the ${PORT} ?

Reply:
oops, typo